roles/validate_inventory/tasks/validate_pxe.yml

```diff
@@ -0,0 +1,33 @@
+---
+- name: Assert 'dns host' and 'tftp host' is have IPs and that is the same
+  assert:
+    that:
+      - hostvars['dns_host']['ansible_host'] is defined
+      - hostvars['tftp_host']['ansible_host'] is defined
+      - hostvars['dns_host']['ansible_host'] == hostvars['tftp_host']['ansible_host']
+    quiet: true
+  when: hostvars['dns_host']['use_pxe'] | default(false)
+
+- name: "Check if ipmitool installed"
```

Comment:
I don't think this is the correct place for these checks. 
They're not checking inventory values. 
Also validate_inventory runs on `localhost` not on the `bastion` which SHOULD be the same but there not guaranteed. 
I think we need a validate_bastion playbook or something like that.

Reply:
Also this needs a check for if any of the nodes have pxe as we only need them installed if pxe is a vendor.

Reply:
Can I add delegate to?

Reply:
> Also this needs a when condition for if any of the nodes have pxe as we only need them installed if pxe is a vendor.

We have this check in the main.yml

Reply:
I don't think `setup_pxe_service` Is the correct value here. We could use an existing pxe server, you need to check the vendor.

Reply:
But if we will add the check that when you have `vendor=pxe` that have to use also the `setup_pxe_service =true` it will be the same not?

Reply:
I added the delegation to bastion